### PR TITLE
Hitbtc as a child class of Bequant

### PR DESCRIPTION
**Pull request by peedrr**
Not sure if you saw message on Slack. I've applied all the new features and authenticated goodness from Bequant to HitBTC as they share the same API. This was achieved by making HItBTC a child of the Bequant class.

For this one, please check over the inheritance.  I know enough to be dangerous but there may well be better ways of doing things, or something that I missed. 

It's only a few lines of code... I threw most of your good work in the bin  🧛‍♂️ 

Have updated the Bequant demo to be a combined Bequant/HitBTC one.

- [X] - Tested
- [X] - Changelog updated
- [ ] - Tests run and pass
- [X] - Flake8 run and all errors/warnings resolved
- [ ] - Contributors file updated (optional)

### CODE REVIEW

File: cryptofeed/exchange/old.hitbtc.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+'''
```
Comment (bmoscon) [inline cryptofeed/exchange/old.hitbtc.py:1]: no reason to live the file, its in the git history

File: cryptofeed/standards.py [hunk 2]
```diff
@@ -115,6 +115,7 @@ def timestamp_normalize(exchange, ts):
     },
     TRADES: {
         DYDX: 'v3_trades',
+        HITBTC: 'subscribeTrades',
```
Comment (bmoscon) [inline cryptofeed/standards.py:118]: Why am I seeing this here? It supported trades before, are you adding a new one and leaving the old entry?

File: cryptofeed/standards.py [hunk 3]
```diff
@@ -267,7 +278,9 @@ def timestamp_normalize(exchange, ts):
     },
     FILL_OR_KILL: {
         BEQUANT: {'timeInForce': 'FOK'},
+        HITBTC: {'timeInForce': 'FOK'},
```
Comment (bmoscon) [inline cryptofeed/standards.py:281]: looks like this was added twice?

File: cryptofeed/exchange/bequant.py [hunk 4]
```diff
@@ -74,6 +73,290 @@ def __reset(self):
         self.l2_book = {}
         self.seq_no = {}
 
+    async def _ticker(self, msg: dict, ts: float):
```
Comment (bmoscon) [inline cryptofeed/exchange/bequant.py:76]: why does it show all this code being added? This code is already present in the repo

**Review by bmoscon — CHANGES_REQUESTED** [review]

File: cryptofeed/exchange/bequant.py (hunk 4 above)
Comment (peedrr) [reply cryptofeed/exchange/bequant.py:76]: Hmm. I think I might know why. I rebased on my 'bequant' branch when I should have rebased on origin (which merged the bequant commits). If you're happy with the changes in principle, reject this one and I'll start afresh on the current master

**Comment by bmoscon** [comment]
@peedrr yes I think the changes in question are fine, will be easier to confirm with out all the duplicated changes. Please restart from master and we'll get it merged

**Pull request closed without merging**